packages/settings-library/src/settings_library/comp_services.py

```diff
@@ -2,7 +2,7 @@
 from pydantic.tools import parse_raw_as
 from settings_library.base import BaseCustomSettings
 
-from .constants import GB
+from ._constants import GB
```

Comment:
why is there a _ in front of the module name? it cannot be re-used outside of the library? stuff like ```GB``` is not really something private is it?

Reply:
It is a way to say to express what is part of the API of the library and what is not
the constants, as you mention, are not too troublesome regarding coupling BUT this way I avoid having the services importing the same constant from different libraries. That is the ida